index.md

```diff
@@ -157,7 +157,7 @@ title: Stanford Systemic Racism Tracker
             <div class="col-1"></div>
             <div class="col-5">
                 <p>The KSR will also help users discover resources about how policing and the criminal justice system shape the ability or inability of African Americans to safeguard their lives and their Constitutional rights, including the right to be “secure in their persons [and] houses,” as articulated by the Fourth Amendment. We chose policing and the courts for the second topic because of the growing interest in and beyond Stanford on how African Americans are seen and treated in the criminal justice system. This concern is at the heart of the Black Lives Matter movement.</p>
-                <p>The KSR will document laws and regulations as well as social movements and the parallel acts of resistance and denial of equal access and equal rights through civil and criminal behaviors, atrocities, and everyday misdeeds (e.g. racial profiling). The purpose of exposing these historical and present-day linkages is to lay the groundwork for further study and teaching as well as research into these attainments and practices as ways to educate the young, students in higher education, researchers, and public policy makers. </p>
+                <p>The KSR will document laws and regulations as well as social movements and the parallel acts of resistance and denial of equal access and equal rights through civil and criminal behaviors, atrocities, and everyday misdeeds (e.g. racial profiling). The purpose of exposing these historical and present-day linkages is to lay the groundwork for further study and teaching as well as research into these attainments and practices as ways to educate the young, students in higher education, researchers, and public policy makers.</p>
             </div>
```

Comment:
This change is just getting rid of an extra empty space before a period -- nothing else changed.